config.php

```diff
@@ -6,5 +6,6 @@
         'style' => 'https://unpkg.com/@rebelcode/bookings-client@0.0.6/dist/static/css/app.css',
         'script' => 'https://unpkg.com/@rebelcode/bookings-client@0.0.6/dist/static/js/app.js',
         'main_script' => plugins_url(RC_WP_BOOKINGS_FRONT_UI_MODULE_RELATIVE_DIR . '/assets/main.js', EDDBK_FILE),
+        'edd_settings' => get_option('edd_settings')

```

Comment:
Later on, this will need to be [centralized](https://github.com/orgs/RebelCode/projects/2#card-10468752).